test/integration/bin/cts/index.test.ts

```diff
@@ -0,0 +1,30 @@
+import { readFile } from 'fs/promises'
+import { createIntegrationTest, deleteFile, existsFile } from '../../utils'
+
+afterEach(async () => {
+  await deleteFile(`${__dirname}/fixtures/tsconfig.json`)
+})
+
+describe('integration', () => {
```

Comment:
better to use different names, like "integration - cts" for `describe()` name, then give it("should ..") to describe the expectation

Reply:
Got it! Do you want me to resolve it on this PR or other?

Reply:
Ye not blocking, can follow up later!